mapiproxy/util/schema_migration.c

```diff
@@ -0,0 +1,151 @@
+/*
+   Schema migration util procedures calling Python code
```

Comment:
No, have nothing to do with that. It is intended to run SQL operations to the following "applications" (OpenChangeDB, Indexing and NamedProperties) using python code